Cargo.toml

```diff
@@ -41,6 +42,8 @@ encoding = "0.2.32"
 failure = "0.1.8"
 mime = "0.3.4"
 
+rsasl = { version = "2.0.0-rc.2", default-features = false, features = ["config_builder"] }
```

Comment:
nit: Why the extra newline above this?

Reply:
The dev-deps are somewhat grouped by functionality, I tried emulating that.